src/ui/agents/BaseAgent.ts

```diff
@@ -8,7 +8,14 @@ export default class BaseAgent implements IBaseAgent {
 
             return Promise.resolve(callback(connection));
         } catch (e) {
-            const msg: string = error || e.message;
+            let msg: string = "";
+            if (error) {
+                msg = error;
+            } else {
+                const errorDetails = ` (${e.response.data.code}: ${e.response.data.message})`;
+                msg = e.message + errorDetails;
+            }
+            console.log(error);
```

Comment:
I think is best to use `console.error` or `console.info` here.

Reply:
The idea is to ensure that this any tool listening to the stream (and any IT person providing support) can identity the source of the problem more easily.

Reply:
done